pkgs/tools/networking/wireguard-go/default.nix

```diff
@@ -17,6 +17,10 @@ buildGoPackage rec {
 
   passthru.updateScript = ./update.sh;
 
+  postInstall = ''
+    ln -sf $out/bin/wireguard $out/bin/wireguard-go
```

Comment:
Not a hard block but the question is how much value backwards compatibility provides given that most people will end up using tools like wg-quick anyway.

Reply:
We shouldn't silently add a compatibility wrapper without notifying users to ever migrate, because then we can't remove the wrapper.

In the case of neomutt, we had a compatibility wrapper nagging users to switch (removed in https://github.com/NixOS/nixpkgs/pull/63687).

In the case of `wireguard-go`, I'd be more in favour of not providing the backwards compat wrapper at all.

Reply:
@Mic92 @flokli fixed.